src/seedu/addressbook/data/person/Phone.java

```diff
@@ -37,23 +35,9 @@ public static boolean isValidPhone(String test) {
     }
 
```

Comment:
You might like to modify your constructor to use `super()`, provided you add a constructor method in `Contact` class. See this [implementation](https://github.com/nus-cs2103-AY1617S1/addressbook-level2/pull/1064/commits/18db3493b65da8e1a7be4fc54ebeff0e2ccf50b9)